BUILD

```diff
@@ -28,7 +28,7 @@ nogo(
         "@org_golang_x_tools//go/analysis/passes/loopclosure:go_default_library",
         "@org_golang_x_tools//go/analysis/passes/lostcancel:go_default_library",
         # "@org_golang_x_tools//go/analysis/passes/nilfunc:go_default_library",
-        "@org_golang_x_tools//go/analysis/passes/nilness:go_default_library",
+        # "@org_golang_x_tools//go/analysis/passes/nilness:go_default_library", # template methods currently cause this analyzer to panic
```

Comment:
have you tried updating the org_golang_x_tools version? it looks like they addressed some generics related issues in newer versions

Reply:
Good idea, I hadn't tried that. Unfortunately, now I have, and the newer version still panics :/